modular_skyrat/modules/modular_items/lewd_items/code/lewd_structures/dancing_pole.dm

```diff
@@ -176,8 +175,7 @@
 		to_chat(user, span_notice("You begin unfastening the frame from the floor and ceiling..."))
 		if(P.use_tool(src, user, 8 SECONDS, volume=50))
 			to_chat(user, span_notice("You disassemble the stripper pole."))
-			var/obj/item/polepack/C = new
-			C.loc = loc
+			new /obj/item/polepack(get_turf(user))
```

Comment:
```suggestion
			new /obj/item/polepack(get_turf(src))
```
perhaps this instead? I don't know, this is just preference at this point, ahaha.